lib/src/tap_outside_detector_widget.dart

```diff
@@ -43,9 +43,12 @@ class _TapOutsideDetectorWidgetState extends State<TapOutsideDetectorWidget> {
 
   void _handleTap(Offset position) {
     if (mounted) {
-      final box = context.findRenderObject() as RenderBox;
-      final localPosition = box.globalToLocal(position);
+      final box = context.findRenderObject();
+      if (box == null || box is! RenderBox) {
+        return;
+      }
```

Comment:
We can probably keep the cast as long as we handle it as nullable.
```suggestion
      final box = context.findRenderObject() as RenderBox?;
      if (box == null) return;
```

In all examples that I found where `findRenderObject()` is called, it's always expected to be non-null value, to the point where it's used with a bang operator `findRenderObject()!`. The question that I have is, how can it be null for this widget?